add how-to-create-a-subproject.md for sig-data-science (#31)
open-services-group/community: sig-data-science/how-to-create-a-subproject.md

# Steps to create a new subproject under SIG-Data-Science:

## Step 1:

> Make sure you have met all the prerequisite needed for a subproject:

Reviewer: ```suggestion
Make sure you have met all the prerequisites needed for a subproject:
```

### Prerequisite:
* A Github repo
* A Github project board
* Project Owners defined in Repo's OWNERS file
* Agreement in an earlier SIG-Data-Science meeting that the subproject should move forward as noted in meeting notes.

## Step 2:

Create a feature branch on your fork of `open-services-group/community` and make changes to the following two files:
* `sig-data-science/README.md`
* `sigs.yaml`

### `sig-data-science/README.md`

> In `sig-data-science/README.md` make an update to the “Details about SIG-Data-Science Subprojetcs” section with the title and a short description of the subproject.

Reviewer 2: ```suggestion
In `sig-data-science/README.md` make an update to the “Details about SIG-Data-Science Subprojects” section with the title and a short description of the subproject.
```

```markdown

> <!-- BEGIN CUSTOM CONTENT -->

Author: The readme is auto generated, so any custom content we want to add independent of the auto-generation process has to be within the CUSTOM CONTENT tags.

> <!-- BEGIN CUSTOM CONTENT -->

Reviewer 3: 👍

> `sig-data-science/README.md`# Details about SIG-Data-Science Subprojects

Author: :+1:  this is a mistake

## Example Subproject 1

Description…

## Example Subproject 2

> Description

Reviewer 2: ```suggestion
Description...
```

<!-- END CUSTOM CONTENT -->

```

### `sigs.yaml`
In `sigs.yaml` under `sigs` → `dir:sig-data-science` find the `meetings` section and add a weekly recurring meeting for the new subproject

```yaml
meetings:
  - description: Data Science SIG meeting
    day: Wednesday
    time: "10:00"
    tz: ET (East Time)
    frequency: bi-weekly
    url: meet.google.com/ufs-hgvi-oni

 - description: Example Subproject
    day: Day of Week
    time: "Time"
    tz: ET (East Time)
    frequency: weekly
    url: google meet link
```

> Under `sigs` → `dir:sig-data-scienc`e find the `subprojects` section and add the new subproject name, description and link to the original repo's OWNERS file.

Reviewer: ```suggestion
Under `sigs` → `dir:sig-data-science` find the `subprojects` section and add the new subproject name, description and link to the original repo's OWNERS file.
```

> Under `sigs` → `dir:sig-data-scienc`e find the `subprojects` section and add the new subproject name, description and link to the original repo's OWNERS file.

Reviewer 3: ```suggestion
Under `sigs` → `dir:sig-data-science` find the `subprojects` section and add the new subproject name, description and link to the original repo's OWNERS file.
```

_NOTE_: The README.md will be auto-generated from this file, so please be sure to add the `description` field as it is seen below, so that it will add the correct link to the description you wrote above in your changes to the README.

```yaml
subprojects:
- name: example-subproject
    description: '[Described below](#example-subproject)'
    owners:
    - https://raw.githubusercontent.com/<subproject-repo>/OWNERS

```

## Step 3:

> Push your changes to GitHub and open a PR in the upstream repo https://github.com/open-services-group/community.

Reviewer 3: ```suggestion
Push your changes to your GitHub fork and open a PR in the upstream repo https://github.com/open-services-group/community.
```
